clj/src/cljd/core.cljd

```diff
@@ -1828,6 +1829,10 @@
   ^:mixin EqualsEquivMixin
   IPrint
   (-print [o ^StringSink sink]
+    (when-let [m (and *print-meta* meta)]
```

Comment:
replace by `when-some`, we don't care about false

Reply:
Because we also need to check if `*print-meta*` is enabled, we do care about false here.